### PR TITLE
Apply post-config tweaks for make CRC target

### PR DESCRIPTION
**Pull request by bogdando**
Apply the post-config CRC commands replicated from OCP Prow CI: https://github.com/openshift/release/pull/34536

### CODE REVIEW

**Comment by raukadah** [comment]
recheck

**Comment by bogdando** [comment]
PTAL

File: devsetup/scripts/crc-setup.sh [hunk 1]
```diff
@@ -71,3 +71,14 @@ sudo update-ca-trust
 if [ $export_path == 1 ]; then
     echo "WARNING: you must add ~/bin in your PATH in order to access to crc binary"
 fi
+
+# apply post-config tweaks
+oc patch etcd cluster -p='{"spec": {"unsupportedConfigOverrides": {"useUnsupportedUnsafeNonHANonProductionUnstableEtcd": true}}}' --type=merge
+oc patch authentications.operator.openshift.io cluster -p='{"spec": {"unsupportedConfigOverrides": {"useUnsupportedUnsafeNonHANonProductionUnstableOAuthServer": true}}}' --type=merge
+oc scale --replicas=1 ingresscontroller/default -n openshift-ingress-operator
+oc scale --replicas=0 deployment.apps/console -n openshift-console
```
Comment (stuggi) [inline devsetup/scripts/crc-setup.sh:79]: I agree, most of the times you probably don't want/need the console, but maybe we want to make this optional. could also to default to disable it

File: devsetup/scripts/crc-setup.sh [hunk 2]
```diff
@@ -71,3 +71,14 @@ sudo update-ca-trust
...
+oc scale --replicas=1 deployment.apps/downloads -n openshift-console
```
Comment (stuggi) [inline devsetup/scripts/crc-setup.sh:80]: isnt' this the default. at least I see this in my env

File: devsetup/scripts/crc-setup.sh [hunk 3]
```diff
@@ -71,3 +71,14 @@ sudo update-ca-trust
...
+oc scale --replicas=1 deployment.apps/oauth-openshift -n openshift-authentication
+oc scale --replicas=1 deployment.apps/packageserver -n openshift-operator-lifecycle-manager
```
Comment (stuggi) [inline devsetup/scripts/crc-setup.sh:82]: same for those two. seems 1 is the default?

File: devsetup/scripts/crc-setup.sh [hunk 4]
```diff
@@ -71,3 +71,14 @@ sudo update-ca-trust
...
+oc scale --replicas=0 deployment.apps/cluster-monitoring-operator -n openshift-monitoring
```
Comment (stuggi) [inline devsetup/scripts/crc-setup.sh:83]: on my 4.13 crc this there is no deployment for this
```
$ oc scale --replicas=0 deployment.apps/cluster-monitoring-operator -n openshift-monitoring
Error from server (NotFound): deployments.apps "cluster-monitoring-operator" not found
```

File: devsetup/scripts/crc-setup.sh [hunk 5]
```diff
@@ -71,3 +71,14 @@ sudo update-ca-trust
 if [ $export_path == 1 ]; then
     echo "WARNING: you must add ~/bin in your PATH in order to access to crc binary"
 fi
+
+# apply post-config tweaks
+oc patch etcd cluster -p='{"spec": {"unsupportedConfigOverrides": {"useUnsupportedUnsafeNonHANonProductionUnstableEtcd": true}}}' --type=merge
+oc patch authentications.operator.openshift.io cluster -p='{"spec": {"unsupportedConfigOverrides": {"useUnsupportedUnsafeNonHANonProductionUnstableOAuthServer": true}}}' --type=merge
```
Comment (stuggi) [inline devsetup/scripts/crc-setup.sh:77]: is there any default tweaking that we should set those?

**Comment by bogdando** [comment]
Closed it as ci-framework already covers this https://github.com/openstack-k8s-operators/ci-framework/pull/327
And we don't need to have it in both places, likely

**Pull request closed without merging**